### PR TITLE
Fetch region from node info [ Do not merge ]

### PR DESCRIPTION
**Pull request by ambiknai**

### CODE REVIEW

File: pkg/ibmcsidriver/controller_helper.go [hunk 1]
```diff
@@ -453,6 +453,7 @@ func createCSIVolumeResponse(vol provider.Volume, capBytes int64, zones []string
 			AccessibleTopology: []*csi.Topology{topology},
 		},
 	}
+	fmt.Println("createVolumeResponse:", volResp)
```
Comment (prankulmahajan) [inline pkg/ibmcsidriver/controller_helper.go:456]: Can we remove this or put in logger ?

File: pkg/ibmcsidriver/controller_helper_test.go [hunk 2]
```diff
@@ -677,6 +647,8 @@ func isCSIResponseSame(expectedVolume *csi.CreateVolumeResponse, actualCSIVolume
 	if expectedVolume == nil || actualCSIVolume == nil {
 		return false
 	}
+	fmt.Println(expectedVolume)
+	fmt.Println(actualCSIVolume)
```
Comment (prankulmahajan) [inline pkg/ibmcsidriver/controller_helper_test.go:651]: Same. Can we remove this? 
Or else required, put in logger

**Review by prankulmahajan — CHANGES_REQUESTED** [review]

File: go.mod [hunk 3]
```diff
@@ -3,7 +3,7 @@ module github.com/kubernetes-sigs/ibm-vpc-block-csi-driver
 go 1.16
 
 require (
```
Comment (sameshai) [inline go.mod:5]: Please changes this later to beta16

**Comment by ambiknai** [comment]
Closing this PR. Delivering the feature via diff approach here https://github.com/kubernetes-sigs/ibm-vpc-block-csi-driver/pull/78

**Pull request closed without merging**